src/emulator/downloadableEmulators.ts

```diff
@@ -51,16 +51,18 @@ const DownloadDetails: { [s in DownloadableEmulators]: EmulatorDownloadDetails }
     },
   },
   gui: {
-    version: "0.0.1",
-    downloadPath: path.join(CACHE_DIR, "gui-v0.0.1.zip"),
-    unzipDir: path.join(CACHE_DIR, "gui-v0.0.1"),
-    binaryPath: path.join(CACHE_DIR, "gui-v0.0.1", `server.bundle.js`),
+    version: "0.0.0",
+    downloadPath: path.join(CACHE_DIR, "gui-vEAP.zip"),
+    unzipDir: path.join(CACHE_DIR, "gui-vEAP"),
+    binaryPath: path.join(CACHE_DIR, "gui-vEAP", `server.bundle.js`),
     opts: {
       cacheDir: CACHE_DIR,
-      remoteUrl: "https://storage.googleapis.com/firebase-preview-drop/emulator/gui-v0.0.1.zip",
-      expectedSize: 1204964,
-      expectedChecksum: "52847b962bb66de639487d96a07a69d3",
+      remoteUrl: "https://storage.googleapis.com/firebase-preview-drop/emulator/gui-vEAP.zip",
+      expectedSize: -1,
+      expectedChecksum: "",
       namePrefix: "gui",
+      skipChecksumAndSize: true,
```

Comment:
In my original comment I meant for `-SNAPSHOT` (or `-EAP`, whatever) to imply `skipChecksumAndSize` and we wouldn't need this field in `EmulatorDownloadDetails` at all.

Reply:
ohhh, so like we'd parse the download URL?

Reply:
Yeah I think so.  `removeOldFiles` (or whatever it's called) already has the logic to do that based on `namePrefix` and stuff.

In most build systems when you see `-SNAPSHOT` it is a signal to not cache anything because the artifact is mutable.

Reply:
I think I prefer having the explicit flags, parsing the URL/version just seems weird to me.